### DISCUSSION TITLE
Debugging on windows

### DISCUSSION THREAD

**Discussion by Tockra** [japaric/discovery#499] [General]
Does somebody know a solution to use the fancy gdb solution which linux provides on windows? 
I tried to use wsl but there I can't find my micro:bit usb device.
Further I tried to just run gdb in wsl but the cargo embed on windows host. But the gdb session is opened at localhost:1337 of my windows host system. I can't get an connection from inside my wsl.

Does somebody have an idea?

**Comment by rrfrse** [comment]
I'm also stuck on this on win10, I can't get the gdb to work, I installed the gcc from the arm link, added the path like instructed. gdb command wont run, gdb doesnt seem installed..

**Reply by rrfrse** [reply]
Download the MinGW installer from the official website: https://sourceforge.net/projects/mingw/

Run the MinGW installer and follow the installation instructions.

During the installation process, make sure to select the "gdb" component by checking the box next to it. You can also select other components you may need.


---------> at least now it seems to be installed
